src/lib/utils.ts

```diff
@@ -44,7 +44,8 @@ export const constants = {
   MAX_EVENT_QUEUE_SIZE: 1000,
   HASH_HEADER_NAME: 'dx-hash',
   SIZE_HEADER_NAME: 'dx-size',
-  LAST_UPDATE_HEADER_NAME: 'dx-last-update'
+  LAST_UPDATE_HEADER_NAME: 'dx-last-update',
+  JSON_PARSER_LIMIT: process.env.JSON_PARSER_LIMIT || '1mb'
```

Comment:
I was looking and today the only env var based configuration is `LOG_LEVEL` and `DATA_DIRECTORY`, and the rest of the configuration is in `config.json`.

Would it make more sense for this to be a `api.jsonBodyLimit` config.json?

If not, maybe a guide (in comments in the code?) would make sense for contributors to know when to add a config parameter as a `config.json` vs. an env var.

Reply:
@peterbroadhurst will be updating this PR to surface the payload size limit in the configuration file, I'll also create a new FireFly issue to ensure that when pinned private messages are batched, their combined payload does not go beyond a configurable limit. FireFly would also need to outright reject single private messages whose payload size go beyond that configurable limit.